sources/packages/web/src/views/aest/Reports.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <full-page-container>
+    <template #header>
+      <header-navigator title="Ministry" subTitle="Reports" />
+    </template>
+    <body-header title="Export financial reports" />
+    <formio
+      formName="exportfinancialreports"
+      @loaded="formLoaded"
+      @submitted="exportReport"
+    ></formio>
+    <v-row class="justify-center m-4"
+      ><v-btn color="primary" @click="submitForm">Export CSV file</v-btn></v-row
+    >
+  </full-page-container>
+</template>
+
+<script lang="ts">
+import { ReportService } from "@/services/ReportService";
+import { ReportsFilterAPIInDTO } from "@/services/http/dto";
+import { useToastMessage } from "@/composables";
+export default {
+  setup() {
+    const toast = useToastMessage();
+    let formData: any = undefined;
+
+    const formLoaded = (form: any) => {
+      formData = form;
+    };
+
+    const submitForm = () => {
+      return formData.submit();
+    };
+
+    const exportReport = async (data: ReportsFilterAPIInDTO) => {
+      try {
+        await ReportService.shared.exportReport(data);
+      } catch (error: unknown) {
```

Comment:
The `error` variable is declared but never used, it can be removed.

Reply:
without error, the signature is incomplete. 
![image](https://user-images.githubusercontent.com/54600590/170297959-785a93ba-f0cc-481e-8751-f596aa4e613a.png)

Reply:
Thought of using _error but since there wasn't any warning, didn't change anything.